opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -151,87 +115,14 @@ message MetricDescriptor {
   // the synchronous instruments) is necessary. It can be used to delegate the
   // aggregation from the application to the agent/collector. See
   // https://github.com/open-telemetry/opentelemetry-specification/issues/617
 
-  // Gauge represents the type of a scalar metric that always exports the
-  // "current value" for every data point. It should be used for an "unknown"
-  // aggregation.
-  // 
-  // A Gauge does not support different aggregation temporalities. Given the
-  // aggregation is unknown, points cannot be combined using the same
-  // aggregation, regardless of aggregation temporalities. Therefore,
-  // AggregationTemporality is not included. Consequently, this also means
-  // "StartTimeUnixNano" is ignored for all data points.
-  //
-  // A Metric of this Type MUST store its values as Int64DataPoint or
-  // DoubleDataPoint.
-  message Gauge {
-    // It describes the value type of the measurement used to build this
-    // aggregation.
-    //
-    // Determines if the points are Int64DataPoint or DoubleDataPoint, as well
-    // as the value type of the exemplars.
-    MeasurementValueType measurement_value_type = 1;
-  }
-
-  // Sum represents the type of a numeric scalar metric that is calculated as a
-  // sum of all reported measurements over a time interval.
-  //
-  // TODO: Decide if this should support only MonotonicSum
-  // https://github.com/open-telemetry/opentelemetry-specification/issues/725.
-  //
-  // A Metric of this Type MUST store its values as Int64DataPoint or
-  // DoubleDataPoint.
-  message Sum {
-    // It describes the value type of the measurement used to build this
-    // aggregation.
-    //
-    // Determines if the points are Int64DataPoint or DoubleDataPoint, as well
-    // as the value type of the exemplars.
-    MeasurementValueType measurement_value_type = 1;
-
-    // aggregation_temporality describes if the aggregator reports delta changes
-    // since last report time, or cumulative changes since a fixed start time.
-    AggregationTemporality aggregation_temporality = 2;
-
-    // If "true" means that the sum is monotonic.
-    bool is_monotonic = 3;
-  }
-
-  // Represents the type of a metric that is calculated by aggregating as a
-  // Histogram of all reported measurements over a time interval.
-  //
-  // A Metric of this Type MUST store its values as HistogramDataPoint.
-  message Histogram {
-    // It describes the value type of the measurement used to build this
-    // aggregation.
-    //
-    // Determines the value type of the exemplars.
-    MeasurementValueType measurement_value_type = 1;
-
-    // aggregation_temporality describes if the aggregator reports delta changes
-    // since last report time, or cumulative changes since a fixed start time.
-    AggregationTemporality aggregation_temporality = 2;
-  }
+  // TODO: Should MeasurementValueType be here instead of in every Aggregation?
```

Comment:
Before merging this PR, I need help to make a decision on this.

Reply:
I agree with Tigdan's point below on this question:
https://github.com/open-telemetry/opentelemetry-proto/pull/202/files#r473095541

Reply:
Moved discussion there.